supply-chain-security/compromises/2018/aur.md

```diff
@@ -0,0 +1,12 @@
+# `acroread` package compromised in AUR
+
+An attacker took over maintenance for the orphaned `acroread` package in Arch Linux's AUR and added code designed to fetch malware from the public internet in user's systems.
+
+## Impact
+
+* System information might have leaked to the attacker
+* Account suspended and commit reverted
+
+## Type of compromise
+
+Orphan package takeover, a type of naming attack.
```

Comment:
Could you add `(Negligence)` here?

Reply:
> Could you add `(Negligence)` here?

Is this negligence or something else?  it kinda fits negligence but i'm not 100% who was negligent.  I don't feel we could specifically point to someone not performing due diligence here.

I feel like this is a situation where we need another definition, especially if we see this occur more often.  Either `(Orphan Package Takeover)` or `(Malicious Adoption)`.  Thoughts?

Reply:
@SantiagoTorres